akka-actor/src/main/scala-2.13-/akka/util/ccompat/package.scala

```diff
@@ -4,10 +4,81 @@
 
 package akka.util
 
+import scala.language.implicitConversions
+import scala.language.higherKinds
+
+import scala.collection.GenTraversable
+import scala.{ collection ⇒ c }
+import scala.collection.generic.{ CanBuildFrom, GenericCompanion, Sorted, SortedSetFactory }
+import scala.collection.{ immutable ⇒ i }
+import scala.collection.{ mutable ⇒ m }
+
 /**
  * INTERNAL API
  *
  * Based on https://github.com/scala/scala-collection-compat/blob/master/compat/src/main/scala-2.11_2.12/scala/collection/compat/PackageShared.scala
  */
-package object ccompat extends ccompat.PackageShared {
+package object ccompat {
```

Comment:
does it have to be public?

Reply:
package objects are always public, but I've reduced the contents to `private[akka]`